src/mono/wasm/runtime/loader/config.ts

```diff
@@ -107,6 +102,9 @@ export async function mono_wasm_load_config(module: DotnetModuleInternal): Promi
 
         normalizeConfig();
 
+        await fetchInitializers(loaderHelpers.config);
```

Comment:
this should not be blocking the rest of the runtime startup, please make it work in parallel.

Reply:
Also it should not be in this file, I think.

Reply:
It is in this file, because of the order of how things should happen

1. Download config
2. Import library initializers
3. Call onConfigLoaded
4. [blazor] Call beforeStart on library initializers

Do you have an idea how to refactor it to support this scenario?